Indiefest Bug Fixes Part 1

BenJLawrence opened the pull request.

BenJLawrence committed:
Fixed some bugs with the cutscene, savemanager and added magic functions

BenJLawrence committed:
update submodule

Colby626 reviewed (changes requested):
The cutscene is totally broken. Hook up stuff in the mainbuildscene

BenJLawrence committed:
Hooked up to build scene

Colby626 committed:
Made the wires less confusing and added wire turn prefab

Colby626 committed:
Merge branch 'post-indiefest-bug-fixes' of https://github.com/EricWRogers/EdgeOfAstral into post-indiefest-bug-fixes

Colby626 committed:
Flushed the airlock doors in the control station, moved jumppad to control station, move stairs to the control station

Colby626 committed:
Merge branch 'post-indiefest-bug-fixes' into post-indiefest-level-edits

BenJLawrence committed:
update submodule

Colby626 committed:
Added colliders to the bookshelves and a patrol point outside the office (makes the office part almost impossible)

Colby626 committed:
Made the holes between floor 2 and 1 bigger and made the monster not be able to get into the office

Colby626 committed:
Lowered player in air movement speed, increased max stamina, made stamina pickups give more, increased monster speed

BenJLawrence committed:
Jump pad, physics mat, stamina pickups w/ physics

temporarily turned off the sound on destruction because I think thats what was causing the crashes

BenJLawrence committed:
Sprint toggle, settings menu jank, lights reset, other things

There are no bugs in Ba Sing Se (No but fr I fixed them all)

BenJLawrence committed:
Forgot changes to my test scene

BenJLawrence committed:
Applied changes to prefabs

BenJLawrence committed:
Merge remote-tracking branch 'origin/post-indiefest-level-edits' into post-indiefest-bug-fixes

BenJLawrence committed:
Prepped main build scene
